src/main/java/com/choelinska/horrorlibz/model/entities/UserEntity.java

```diff
@@ -0,0 +1,14 @@
+package com.choelinska.horrorlibz.model.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity(name = "User")
+public class UserEntity {
+    @Id
+    private long id;
```

Comment:
Would be great to have the autoincrementing id, as I can't imagine we will receive it in the request - the user doesn't know the contents of our database to enter the value himself. You can try to use the @GeneratedValue annotation. You should enter a chosen strategy as parameter to it.